hedera-node/src/main/java/com/hedera/services/state/submerkle/ExpirableTxnRecord.java

```diff
@@ -210,6 +218,7 @@ public void serialize(SerializableDataOutputStream out) throws IOException {
 
 		out.writeSerializableList(tokens, true, true);
 		out.writeSerializableList(tokenAdjustments, true, true);
+		out.writeSerializableList(nftTokenAdjustments, true, true);
```

Comment:
We must serialize new properties last. If previous versions of expirable tx record try to process the new version during sync (syncing previous/historical transactions), it will throw error.

We must serialise the same order we deserialise and we deserialse the nftTokenAdjustments last **only if version is 0150**

Reply:
Corrected!